src/OceanSeaIceModels/CrossRealmFluxes/atmosphere_ocean_fluxes.jl

```diff
@@ -365,9 +365,9 @@ end
     time = Time(clock.time)
 
     @inbounds begin
-        Tₒ = ocean_temperature[i, j, 1]
+        Tₒ = ocean_temperature[i, j, kᴺ]
```

Comment:
this was definitely one issue for getting warm SSTs!

this PR is from long ago though... not part of our current problems

but there was indeed a bug because the bulk formulae was seeing the temperature at the ocean floor instead of the one at the ocean surface and that was resulting in much much more heat fluxes

cc @taimoorsohail